README.md

```diff
@@ -116,61 +154,77 @@ for _, lsp in ipairs(servers) do
 end
 EOF
 ```
+
 The `on_attach` hook is used to only activate the bindings after the language server attaches to the current buffer.
 
 ## Debugging
-The two most common reasons a language server does not start or attach are:
 
-1. The language server is not installed. nvim-lspconfig does not install language servers for you. You should be able to run the `cmd` defined in each server's lua module from the command line and see that the language server starts. If the `cmd` is an excutable name, ensure it is on your path.
+The two most common reasons a language server does not start or attach are:
 
-2. Not triggering root detection. The language server will only start if it is opened in a directory, or child directory, containing a file which signals the *root* of the project. Most of the time, this is a `.git` folder, but each server defines the root config in the lua file. See [CONFIG.md](CONFIG.md) or the source for the list of root directories.
+1. The language server is not installed.
+   nvim-lspconfig does not install language servers for you.
+   You should be able to run the `cmd` defined in each server's lua module from the command line and see that the language server starts.
+   If the `cmd` is an executable name, ensure its location is listed in your system's `PATH` variable.
+1. Not triggering root detection.
+   The language server will only start if it is opened in a directory, or child directory, containing a file which signals the _root_ of the project.
+   Most of the time, this is a `.git` folder, but a server may define the root differently: see [CONFIG.md](CONFIG.md) or the source for the list of root directories.
 
-:LspInfo provides a handy overview of your active and configured language servers.  Note, that it will not report any configuration changes applied in `on_new_config`.
+`:LspInfo` provides a handy overview of your active and configured language servers.
+However, `:LspInfo` can be inaccurate if any configuration changes were applied in the `on_new_config()` hook.
```

Comment:
Inaccurate is a strong word, I like the old wording better-ish

`:LspInfo` will not report any configuration changes applied in the `on_new_config` hook.

Reply:
Sounds good to me.